package.json

```diff
@@ -67,39 +67,43 @@
     "mime-types": "^2.0.8",
     "once": "^1.3.1",
     "pumpify": "^1.3.3",
+    "safe-buffer": "^5.1.1",
     "stream-events": "^1.0.1",
     "string-format-obj": "^1.0.0",
     "through2": "^2.0.0"
   },
   "devDependencies": {
     "@google-cloud/nodejs-repo-tools": "^2.0.0-beta.8",
-    "ava": "^0.22.0",
     "codecov": "^2.3.0",
+    "eslint": "^4.7.1",
+    "eslint-config-prettier": "^2.5.0",
+    "eslint-plugin-node": "^5.1.1",
+    "eslint-plugin-prettier": "^2.3.1",
     "ink-docstrap": "https://github.com/docstrap/docstrap/tarball/master",
     "intelli-espower-loader": "^1.0.1",
     "jsdoc": "^3.5.4",
     "mocha": "^3.5.3",
     "normalize-newline": "^3.0.0",
     "nyc": "^11.1.0",
     "power-assert": "^1.4.4",
+    "prettier": "^1.7.0",
     "prop-assign": "^1.0.0",
     "propprop": "^0.3.0",
-    "proxyquire": "^1.7.10",
+    "proxyquire": "^1.8.0",
     "request": "^2.70.0",
     "semistandard": "^11.0.0",
-    "sinon": "^3.2.1",
     "tmp": "^0.0.33",
-    "uuid": "^3.0.1"
+    "uuid": "^3.1.0"
   },
   "scripts": {
     "docs": "node_modules/.bin/jsdoc -c .jsdoc.js",
-    "lint": "npm run samples-lint",
+    "lint": "eslint src/ samples/ system-test/ test/",
```

Comment:
Note that we don't actually have to modify repo-tools itself to change the linter. We can just do the following:

1. Add the following to `.cloud-repo-tools.json`:

```
"lint": {
  "cmd": "eslint"
}
```

2. Change the `lint` script to the following:

```
"lint": "repo-tools lint src/ samples/ system-test/ test/"
```